src/Core/Managed/Shared/Extensibility/Implementation/Metrics/MetricTerms.cs

```diff
@@ -0,0 +1,77 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Metrics
+{
+    using System;
+
+    internal static class MetricTerms
+    {
+        private const string MetricPropertiesNamePrefix = "Microsoft.ApplicationInsights.Metrics";
+
+        public static class Aggregation
+        {
+            public static class Interval
+            {
+                public static class Moniker
+                {
+                    public const string Key = MetricPropertiesNamePrefix + ".Aggregation.IntervalMs";
+                }
+            }
+        }
+
+        public static class Extraction
+        {
+            public static class ProcessedByExtractors
+            {
+                public static class Moniker
+                {
+                    public const string Key = MetricPropertiesNamePrefix + ".Extraction.ProcessedByExtractors";
+                    public const string ExtractorInfoTemplate = "(Name:{0}, Ver:{1})";      // $"(Name:{ExtractorName}, Ver:{ExtractorVersion})"
+                }
+            }
+        }
+
+        public static class Autocollection
+        {
+            public static class Moniker
+            {
+                public const string Key = MetricPropertiesNamePrefix + ".MetricIsAutocollected";
```

Comment:
If this propagates all the way to AI Endpoint, we will count this huge string towards the bill. We might want to reduce the footprint if it indeed does.